index.ts

```diff
@@ -163,14 +163,48 @@ collect.set('isNotifications', [
 
 export const isOrganizationProfile = (): boolean => exists('meta[name="hovercard-subject-tag"][content^="organization"]');
 
+export const isOrganizationRepoList = (url: URL | HTMLAnchorElement | Location = location): boolean => /^orgs\/[^/]+\/repositories$/.test(getCleanPathname(url));
+collect.set('isOrganizationRepoList', [
+	'https://github.com/orgs/github/repositories',
+	'https://github.com/orgs/github/repositories?page=2',
+]);
+
 export const isOrganizationRepo = (): boolean => Boolean(document.querySelector<HTMLElement>('[data-owner-scoped-search-url]')?.dataset['ownerScopedSearchUrl']!.startsWith('/org'));
 
-export const isOrganizationDiscussion = (url: URL | HTMLAnchorElement | Location = location): boolean => /^orgs\/[^/]+\/teams\/[^/]+($|\/discussions)/.test(getCleanPathname(url));
-collect.set('isOrganizationDiscussion', [
+export const isOrganizationPackagesList = (url: URL | HTMLAnchorElement | Location = location): boolean => /^orgs\/[^/]+\/packages$/.test(getCleanPathname(url));
+collect.set('isOrganizationPackagesList', [
+	'https://github.com/orgs/github/packages',
+]);
+
+export const isOrganizationPeopleList = (url: URL | HTMLAnchorElement | Location = location): boolean => /^orgs\/[^/]+\/people$/.test(getCleanPathname(url));
+collect.set('isOrganizationPeopleList', [
+	'https://github.com/orgs/github/people',
+	'https://github.com/orgs/github/people?page=2',
+]);
+
+export const isOrganizationProjectsList = (url: URL | HTMLAnchorElement | Location = location): boolean => /^orgs\/[^/]+\/projects$/.test(getCleanPathname(url));
+collect.set('isOrganizationProjectsList', [
+	'https://github.com/orgs/github/projects',
+]);
+
+export const isOrganizationProject = (url: URL | HTMLAnchorElement | Location = location): boolean => /^orgs\/[^/]+\/projects\/[^/]+$/.test(getCleanPathname(url));
+collect.set('isOrganizationProject', [
+	'https://github.com/orgs/cncf/projects/1',
+]);
+
+export const isOrganizationDiscussionsList = (url: URL | HTMLAnchorElement | Location = location): boolean => /^orgs\/[^/]+\/discussions$/.test(getCleanPathname(url));
+collect.set('isOrganizationDiscussionsList', [
+	'https://github.com/orgs/github/discussions',
+]);
+
+export const isOrganizationTeamDiscussion = (url: URL | HTMLAnchorElement | Location = location): boolean => /^orgs\/[^/]+\/teams\/[^/]+($|\/discussions)/.test(getCleanPathname(url));
+collect.set('isOrganizationTeamDiscussion', [
 	'https://github.com/orgs/refined-github/teams/core-team/discussions?pinned=1',
 	'https://github.com/orgs/refined-github/teams/core-team/discussions/1',
 	'https://github.com/orgs/refined-github/teams/core-team',
 ]);
+/** @deprecated use isOrganizationTeamDiscussion instead */
+export const isOrganizationDiscussion = isOrganizationTeamDiscussion;
```

Comment:
Tests are failing because this still needs example URLs. I don't remember how to copy them, but I think this happened before so you can look in this file

Reply:
Yes, I copied from `50bd30f` which did not take URLs arguments ( something I realized only after I looked more closely after seeing the test fail ).